v3-nginx/Dockerfile-alpine

```
@@ -50,7 +50,8 @@ RUN set -eux; \
     ./build.sh; \
     git submodule init; \
     git submodule update; \
-    ./configure --with-yajl --with-ssdeep --with-lmdb --with-geoip --with-pcre2; \
+    sed -i -e 's/^PCRE2_POSSIBLE_LIB_NAMES=\(.*\)"/PCRE2_POSSIBLE_LIB_NAMES=\1 libpcre2-8"/' build/pcre2.m4 ; \
```

Not sure this works: use `$(pcre2-config --prefix)` in both instances, instead of hard coding `libpcre2-8`.
```suggestion
    # patch make file to support Debian package names (see https://github.com/SpiderLabs/ModSecurity/issues/2810) \
    sed -i -e "s/^PCRE2_POSSIBLE_LIB_NAMES=\(.*\)\"/PCRE2_POSSIBLE_LIB_NAMES=\1 $(pcre2-config --prefix)\""/ build/pcre2.m4; \
```